src/include/txn.h

```diff
@@ -177,13 +177,26 @@ struct __wt_txn_op {
 		WT_TXN_OP_NONE,
 		WT_TXN_OP_BASIC,
 		WT_TXN_OP_INMEM,
+		WT_TXN_OP_BASIC_ROW,
+		WT_TXN_OP_INMEM_ROW,
```

Comment:
These new types are necessary to identify the difference between a row store update and a column store update (since they are stored in a union) when deciding whether to free the key.